### ISSUE TITLE
Duplicating a Job Listing Breaks Multiple File Upload Fields

### ISSUE THREAD

**Issue by spencerfinnell** [Automattic/WP-Job-Manager#732]
#### What I Expected To Happen

Duplicate a job listing that has a file upload field set to `multiple` and all images be copied.
#### What Happened Instead

The images do not appear copied.
#### Steps to Reproduce the Bug
1. Set a file upload field to accept multiple images.
2. Submit a listing with multiple images in the field.
3. Duplicate the listing.
#### Recommended Fix

Unserialize any data before sending it back through `update_post_meta`

`update_post_meta( $new_post_id, $meta_key, maybe_unserialize( $meta_value ) );`

https://github.com/Automattic/WP-Job-Manager/blob/master/wp-job-manager-functions.php#L813

**Comment by chaselivingston** [comment]
@spencerfinnell How are you adding multiple file upload fields exactly? Is it the Listing Gallery feature in Listify?

**Comment by spencerfinnell** [comment]
Yes.

Just filtering the fields:

```
public function gallery_images( $fields ) {
	$fields[ 'job' ][ 'gallery_images' ] = array(
		'label'       => __( 'Gallery Images', 'listify' ),
		'type'        => 'file',
		'multiple'    => true,
		'required'    => false,
		'placeholder' => '',
		'priority'    => 4.999,
		'ajax'        => true,
		'allowed_mime_types' => array(
			'jpg'  => 'image/jpeg',
			'jpeg' => 'image/jpeg',
			'gif'  => 'image/gif',
			'png'  => 'image/png'
		)
	);
	return $fields;
}
```

**Comment by tripflex** [comment]
Users can also add multiple upload fields by using my field editor plugin as well, which there are probably at least 500+ doing so already (if not more) ... using clone, probably not, but custom multiple file upload fields, yes.

**Comment by spencerfinnell** [comment]
Definitely very common. Not theme specific. Any multi upload field will fail on duplication.

**Comment by chaselivingston** [comment]
@spencerfinnell @tripflex Thanks for the information!

**Comment by turtlepod** [comment]
Hi All,
How to duplicate a listing. I don't think WPJM have this functionality.
Is it using a plugin? like https://wordpress.org/plugins/post-duplicator/ ?

**Comment by jom** [comment]
Fixed in #911
